### PR TITLE
Add Pagination feature

### PR DESCRIPTION
**Pull request by wuqiten**
1. Add continuationToken as optional parameter for ListRelationships/ListIncomingRelationships/ListModels API;
2. Add continuationToken Handler in the script;
3. Fix typo;
4. Remove updatenextlink policy;

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/AzureDigitalTwins/apiProperties.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
           "redirectUrl": "https://global.consent.azure-apim.net/redirect",
           "properties": {
             "IsFirstParty": "True",
-            "AzureActiveDirectoryResourceId": "https://digitaltwins.azure.net"
+            "AzureActiveDirectoryResourceId": "https://digitaltwins.azure.net",
+            "IsOnbehalfofLoginSupported": true
```
Comment (sankumrav) [inline certified-connectors/AzureDigitalTwins/apiProperties.json:26]: Austin, this property is not required.

File: certified-connectors/AzureDigitalTwins/apiProperties.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
       "adtInstance": {
         "type": "string",
         "uiDefinition": {
-          "displayName": "ADT Instance Name:",
+          "displayName": "ADT Instance name",
```
Comment (sankumrav) [inline certified-connectors/AzureDigitalTwins/apiProperties.json:7]: Lets leave it the way it was before.

File: certified-connectors/AzureDigitalTwins/apiProperties.json (hunk 1 above)
Comment (wuqiten) [reply certified-connectors/AzureDigitalTwins/apiProperties.json]: Fixed

File: certified-connectors/AzureDigitalTwins/apiProperties.json (hunk 2 above)
Comment (wuqiten) [reply certified-connectors/AzureDigitalTwins/apiProperties.json]: Fixed

**Review by Amjed-Ayoub — APPROVED** [review]

File: certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1303,7 +1313,13 @@
       "description": "The id of the relationship. The id is unique within the digital twin and case sensitive.",
       "x-ms-summary": "Relationship id"
     },
-    "addValueTemplate":{
+    "continuationToken": {
+      "name": "continuationToken",
+      "in": "query",
+      "required": false,
+      "type": "string"
```
Comment (Amjed-Ayoub) [inline certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json:1320]: @wuqiten The 'x-ms-summary' property is required, and the 'description' property is required. Please add both.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by wuqiten** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by wuqiten** [comment]
@microsoft-github-policy-service agree company=Microsoft

**Review by Amjed-Ayoub — APPROVED** [review]

**Pull request merged**